### ISSUE TITLE
HCY not exposed

### ISSUE THREAD

**Issue by bennyschudel** [scijs/color-space#28]
Is there a reason, why hcy is not exposed in index.js?

**Comment by dfcreative** [comment]
Yeah, because it is not tested yet, no guarantees it will work properly.

**Comment by dy** [comment]
HCY is exposed in **v3** (`space.hcy`), reimplemented as Chilliant luma-based HCY (Y = Rec.601 luma) with an achromatic guard and exact round-trip. Will close when v3 (#67) merges.
